From c71dbde208aaafaa00292161b63ebbcf7480c831 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Wed, 1 Apr 2015 15:04:05 +0300
Subject: [PATCH] Baytrail does not have core C7 MSR to read

So do not acquire core C7 from MSR, or else it fails
and stops the program.

Reported-by: Eoff, Ullysses A <ullysses.a.eoff@intel.com>
Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 src/cpu/intel_cpus.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/cpu/intel_cpus.cpp b/src/cpu/intel_cpus.cpp
index 167f1a7..e019484 100644
--- a/src/cpu/intel_cpus.cpp
+++ b/src/cpu/intel_cpus.cpp
@@ -131,11 +131,6 @@ nhm_core::nhm_core(int model)
 	if (model == 0x37) {
 		has_c3_res = 0;
 		has_c1_res = 1;
-		this->byt_has_ahci();
-                if ((this->get_byt_ahci_support()) == 0)
-                        has_c7_res = 1;/*BYT-T PC7 <- S0iX*/
-                else
-                        has_c7_res = 0;
 	} else {
 		has_c3_res = 1;
 		has_c1_res = 0;
-- 
2.21.0

